### PR TITLE
Add key activity query

### PR DESCRIPTION
**Pull request by tifrel**

### CODE REVIEW

File: packages/data/src/api/keyActivity/keyActivity.test.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    expect(data?.keyActivity.length).toBe(2);
+    expect(data?.keyActivity[0].kind).toBe('add');
+    expect(data?.keyActivity[1].kind).toBe('del');
+    expect(data?.keyActivity[0].receiptId).toBe('receiptIdAdd');
+    expect(data?.keyActivity[1].receiptId).toBe('receiptIdDel');
+    expect(data?.keyActivity[0].publicKey).toBe('pubkey');
+    expect(data?.keyActivity[1].publicKey).toBe('pubkey');
+    expect(data?.keyActivity[0].timestamp).toBe('2000-01-01T00:00:00.000000');
+    expect(data?.keyActivity[1].timestamp).toBe('2000-01-02T00:00:00.000000');
```
Comment (bh2smith) [inline packages/data/src/api/keyActivity/keyActivity.test.ts]: Could you just make a json object equality on this?

**Review by bh2smith — APPROVED** [review]

File: packages/data/src/api/keyActivity/keyActivity.test.ts (hunk 1 above)
Comment (tifrel) [reply packages/data/src/api/keyActivity/keyActivity.test.ts]: [Not according to the docs](https://jestjs.io/docs/expect#tobevalue)

Comment (bh2smith) [reply packages/data/src/api/keyActivity/keyActivity.test.ts]: You could json.stringify

Comment (tifrel) [reply packages/data/src/api/keyActivity/keyActivity.test.ts]: Made it more concise though by using `JSON.stringify`

**Review by sainthiago — APPROVED** [review]

**Pull request merged**